apps/accelerate/chatllama/chatllama/llama_model.py

```diff
@@ -40,16 +97,314 @@ def __call__(self, texts: Union[List[str], str], *args, **kwargs):
             )
             for i, text in enumerate(texts):
                 tokens[i, : len(text)] = torch.tensor(text).cuda().long()
+
```

Comment:
Instead of creating the left_tokens can't you just modify the line above?

Something like
```python
tokens[i, -len(text):] = torch.tensor(text).cuda().long()
```
should do the trick, right?